src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -181,6 +181,34 @@ def __init__(
         self.vae_scale_factor = 2 ** (len(self.vae.config.block_out_channels) - 1)
         self.register_to_config(requires_safety_checker=requires_safety_checker)
 
+    def load_textual_inversion_embeddings(self, embeddings):
+        r"""
+        Loads textual inversion embeddings. Receives a dictionary with the following keys:
+        - `token`: name of the token to be added to the tokenizers' vocabulary
+        - `embedding`: path to the embedding of the token to be added to the text encoder's embedding matrix
+
+        Iters through the dictionary and adds the token to the tokenizer's vocabulary and the embedding to the text
```

Comment:
Also, we could directly accept a path to the embeddings along with the token rather than dict. The token can be optional; if it's provided, we use that token; otherwise, we use the token present in the `embeddings`

Reply:
Done. Thank you for the suggestion!